folium/folium.py

```diff
@@ -30,6 +30,23 @@ class Map(_Map):
     """This class inherits from the map.Map object in order to provide bindings to
     former folium API.
     """
+    def create_map(self, path='map.html', plugin_data_out=True, template=None):
+        """Write Map output to HTML.
+
+        Parameters:
+        -----------
+        path: string, default 'map.html'
+            Path for HTML output for map
+        plugin_data_out: boolean, default True
+            Deprecated, not used anymore
+        template: string, default None
+            Deprecated, not used anymore
+        """
+        warnings.warn("%s is deprecated. Use %s instead" % ("Map.create_map",
```

Comment:
Awesome!  I was going to suggest that.  I like `save` or `save_map` better than `create_map`.